src/api/helpers/send-email-application-submission-deadline/index.test.ts

```diff
@@ -0,0 +1,104 @@
+import applicationSubmissionDeadlineEmail from '.';
+import applicationSubmissionDeadineEmail from './send-email';
+import sendEmail from '../../emails';
+import getKeystoneContext from '../../test-helpers/get-keystone-context';
+import { createFullApplication } from '../../test-helpers';
+import { mockSendEmailResponse } from '../../test-mocks';
+import { DATE_24_HOURS_FROM_NOW } from '../../constants';
+import { dateInTheFutureByDays } from '../date';
+import applications from '../../test-helpers/applications';
+import { Application, Context } from '../../types';
+
+describe('helpers/send-email-application-submission-deadline', () => {
+  let context: Context;
+  let application: Application;
+
+  jest.mock('../../emails');
+  jest.mock('./send-email');
+
+  let sendEmailSubmissionDeadlineSpy = jest.fn();
+  let sendEmailSpy = jest.fn();
+
+  const today = new Date();
+  const twoDaysInFuture = dateInTheFutureByDays(today, 2);
```

Comment:
Throughout this file and in test descriptions, could 2 be using `REMINDER_DAYS` constant?

So that if the days change, e.g to 4, this will automatically update